reposerver/repository/repository.go

```diff
@@ -680,7 +680,7 @@ func (s *Service) GetAppDetails(ctx context.Context, q *RepoServerAppDetailsQuer
 		res.Kustomize = &KustomizeAppSpec{}
 		res.Kustomize.Path = q.Path
 		k := kustomize.NewKustomizeApp(appPath, newCreds(q.Repo))
-		_, imageTags, images, err := k.Build(nil)
+		_, imageTags, images, err := k.Build(nil, nil)
```

Comment:
nil?

Reply:
Fixed. Had to add `server.settings.SettingsManager` to `server.repository.Server`